flow360/cloud/requests.py

```diff
@@ -28,6 +28,22 @@ class Config:  # pylint: disable=too-few-public-methods
         allow_population_by_field_name = True
 
 
+class NewSurfaceMeshRequest(Flow360Requests):
+    """request for new surface mesh"""
+
+    name: str = pd.Field(alias="name")
+    stem: str = pd.Field(alias="stem")
+    tags: Optional[List[str]] = pd.Field(alias="tags")
+    geometry_id: Optional[str] = pd.Field(alias="geometryId")
+    config: Optional[str] = pd.Field(alias="config")
+    mesh_format: Literal["aflr3", "cgns", "stl"] = pd.Field(alias="meshFormat")
+    endianness: Optional[Literal["little", "big"]] = pd.Field(alias="meshEndianness")
+    compression: Optional[Literal["gz", "bz2", "zst"]] = pd.Field(alias="meshCompression")
```

Comment:
do we need to send this information to backend? Can you check with Yuwei or Juncheng?

Reply:
yes for now, but this will be refactored in the future so that the backend server has no knowledge on this level of details. The plan has just started.